src/utils/logger.ts

```diff
@@ -0,0 +1,71 @@
+import { getExtensionLogger, IVSCodeExtLogger } from "@vscode-logging/logger";
+import { ExtensionContext } from "vscode";
+import { getConfig as internalGetConfig } from "../config";
+import { VSCodeAbstraction } from "../types";
+import { getExtensionId, getExtensionName } from "./getExtensionId";
+
+let _logger: IVSCodeExtLogger;
```

Comment:
Was bedeutet der Unterstrich vor der Variable in dieser Datei? Wenn wir diesen Syntax einführen, sollten wir es konsistenterweise überall einführen, sonst finde ich es etwas verwirrend. Beispielsweise gibt es in der Datei "getExtensionId.ts" ebenfalls globale Variablen.

Reply:
Guter Punkt. 

Nur eine kleine Anmerkung: Die Variable gibts nur innerhalb des Modules und nicht global.